lib/launchdarkly_flutter.dart

```diff
@@ -72,26 +168,46 @@ class LaunchdarklyFlutter {
   /// 5 seconds, it is returned anyway and can be used, but may not
   /// have fetched the most recent feature flag values.
   /// [mobileKey] is the mobile key from your Environments page in LaunchDarkly.
+  /// Additional configuration can be set via the optional [config].
   /// [userKey] is the user id considered by LaunchDarkly for feature flag targeting and rollouts.
   /// The userKey should also uniquely identify each user. You can use a primary key, an e-mail address,
   /// or a hash, as long as the same user always has the same key. We recommend using a hash if possible.
   /// You can also distinguish logged-in users from anonymous users in the SDK by leaving the userKey parameter null.
-  /// You can pass custom arguments in [custom] map.
+  /// You can pass built-in user attributes as [LaunchDarklyUser] in [user].
+  /// You can pass custom attributes and private custom attributes in [custom] and [privateCustom] maps accordingly.
+  /// Please note private attributes take precedence over non-private ones.
   Future<bool?> init(
     String? mobileKey,
     String? userKey, {
+    LaunchDarklyConfig? config,
+    LaunchDarklyUser? user,
     Map<String, dynamic>? custom,
+    Map<String, dynamic>? privateCustom,
   }) async {
     if (userKey == null) {
       return await _channel.invokeMethod('init', <String, dynamic>{
         'mobileKey': mobileKey,
-        'custom': custom,
+        'config': config?.toMap(),
```

Comment:
Why is it that we are not treating `user` in this conditional, only on the else?

Reply:
I thought if `userKey` is missing then `user` is likely to miss as well. But you're right it's better not to make this assumption. Thank you for the note.